src/components/views/profiles/profiles.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from "react";
+
+import './profiles.style.scss';
+
+class UserCards extends Component {
```

Comment:
Let's go with functional components and get rid of class components.
Also use hooks instead state. - useState, useEffect